src/model/delta/delta.js

```diff
@@ -57,6 +57,15 @@ export default class Delta {
 	}
 
 	/**
+	 * @param {Number} baseVersion
+	 */
+	set baseVersion( baseVersion ) {
+		for ( let operation of this.operations ) {
+			operation.baseVersion = baseVersion++;
```

Comment:
Okay, I'll change it but in next iteration of changes for Undo this code will be removed anyway.